frontend/packages/shared/src/components/GiteaHeader/ThreeDotsMenu/LocalChangesModal/LocalChangesModal.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactNode } from 'react';
+import classes from './LocalChangesModal.module.css';
+import { Heading } from '@digdir/design-system-react';
+import { MonitorIcon } from '@navikt/aksel-icons';
+import { StudioModal } from '@studio/components';
+import { useTranslation } from 'react-i18next';
+import { LocalChanges } from './LocalChanges/LocalChanges';
+
+export type LocalChangesModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  org: string;
+  app: string;
+};
+
+export const LocalChangesModal = ({
+  isOpen,
+  onClose,
+  org,
+  app,
+}: LocalChangesModalProps): ReactNode => {
+  const { t } = useTranslation();
+
+  return (
+    <StudioModal
+      isOpen={isOpen}
+      onClose={onClose}
+      title={
+        <div className={classes.headingWrapper}>
+          <MonitorIcon className={classes.icon} />
+          <Heading level={1} size='small'>
+            {t('sync_header.local_changes')}
+          </Heading>
+        </div>
+      }
+    >
+      <div className={classes.localChangesContent}>
```

Comment:
Hva er hensikten med denne wrapperen?

Reply:
Hvis det er for å fikse feilen som @standeren påpekte, bør vi heller fikse det inne i `StudioModal`-komponenten. Å sette `overflow: auto` på rotklassen bør være tilstrekkelig.

Reply:
har lagt til `overflow `i StudioModal ,men vi fortsatt trenger `localChangesContent` for at content ikke går under header som den ser ut i vidoe

https://github.com/Altinn/altinn-studio/assets/90609090/53290f23-4305-4b34-8ea6-69711b4053db

Reply:
Vi må så langt det lar seg gjøre ta tak i problemene der de kommer fra, ikke løse dem ved å overstyre kode i flere lag. Da ender vi opp med veldig uoversiktlig kode.
Feilen du får tyder igjen på at det er noe rart med CSS-en i `StudioModal`, så vi må løse problemet der. Jeg har tatt en titt og ser at det er en del merkelig bruk av `position` der. Dette er ting som enkelt lar seg løse med `flex`. Jeg foreslår at vi gjør følgende endringer i `StudioModal`:

I `StudioModal.tsx`, legg til en `div` med en egen klasse rundt tittelen og fjern wrapperen rundt sletteknappen:
```
        <div className={classes.headingWrapper}>
          <div className={classes.title}>
            {title}
          </div>
          <Button
            icon={<MultiplyIcon />}
            onClick={onClose}
            title={t('modal.close_icon')}
            variant='tertiary'
          />
        </div>
```
Det gjør at vi kan sette flex-egenskaper på den i CSS-en. `closeButtonWrapper` vil vi ikke ha behov for lengre.

Så kan du endre hele `StudioModal.module.css` slik:
```
.modal {
  --modal-min-width: 320px;
  --modal-min-height: 100px;
  --modal-max-height: 80vh;
  --modal-max-width: 80%;
  --modal-position: 50%;
  --modal-translate: calc(var(--modal-position) * -1);
  --heading-height: 60px;
  --modal-border-size: 1px;
  --modal-border-radius: var(--fds-border_radius-medium);
  --modal-content-max-height: calc(var(--modal-max-height) - var(--heading-height)) - var(--modal-border-size);
  --modal-padding: var(--fds-spacing-4);

  background-color: var(--fds-semantic-background-default);
  border-radius: var(--modal-border-radius);
  border: var(--modal-border-size) solid var(--fds-semantic-border-neutral-subtle);
  display: flex;
  flex-direction: column;
  height: max-content;
  left: var(--modal-position);
  max-height: var(--modal-max-height);
  max-width: var(--modal-max-width);
  min-height: var(--modal-min-height);
  min-width: var(--modal-min-width);
  position: absolute;
  top: var(--modal-position);
  transform: translate(var(--modal-translate), var(--modal-translate));
  width: max-content;
}

.modalOverlay {
  background-color: rgba(30, 43, 60, 0.5);
  bottom: 0;
  left: 0;
  position: fixed;
  right: 0;
  top: 0;
  z-index: 1000;
}

.headingWrapper {
  align-items: center;
  background-color: var(--fds-semantic-background-default);
  border-bottom: 1px solid var(--fds-semantic-border-divider-default);
  display: flex;
  padding: var(--modal-padding);
}

.contentWrapper {
  flex: 1;
  overflow: auto;
  padding: var(--modal-padding);
}

.title {
  flex: 1;
}
```
Hovedendringen her er at jeg har fjernet `position` fra `closeButtonWrapper` og `headingWrapper`. Så har jeg tatt i bruk `flex` og satt en konsistent `padding` (`--modal-padding`) rundt hele. Dette gjør at vi heller ikke trenger å sette dette spesifikt der modalen brukes, og vi får samme avstand i alle modalene.

Reply:
pga denne linje har `  --modal-padding: var(--fds-spacing-4);`  Cypress funket ikke i to steder, jeg endret del til 3 så kjørte Cypress grønnt